external/deep-object-reid/torchreid_tasks/openvino_task.py

```diff
@@ -200,6 +201,28 @@ def infer(self, dataset: DatasetEntity,
             update_progress_callback(int(i / dataset_size * 100))
         return dataset
 
+    @check_input_parameters_type({"dataset": DatasetParamTypeCheck})
+    def explain(self, dataset: DatasetEntity,
```

Comment:
I propose to support `explain` for OV task in a different PR. For this PR let's support only torch inference.
In addition, it is required to consider the OV task support in the new otx at `feature/otx` branch. It is better to build `explain` on top of `feature/otx` rather than on top of backends.

Reply:
Yes, then let's deal it with another PR.